draft-rha-jose-hpke-encrypt.md

```diff
@@ -199,6 +200,14 @@ The recipient will create the receiving HPKE context by invoking SetupBaseR() (S
 
 The Open function will, if successful, decrypts "ct".  When decrypted, the result will be either the CEK (when Key Encryption mode is used), or the content (if Integrated Encryption mode is used).  The CEK is the symmetric key used to decrypt the ciphertext.
 
+## Protected Header Parameters
+
+In order to ensure the security properties of HPKE, the following requirements MUST be followed for protected header parameters:
+
+* For Integrated Encryption, "epk" MUST be present, and MUST contain an Encapsulated JSON Web Key.
+* For Integrated Encryption, "alg" MUST be "HPKE-IntEnc", as defined in {{IANA}}, and "enc" MUST be an HPKE algorithm from JSON Web Signature and Encryption Algorithms in {{JOSE-IANA}}.
+* For Key Encryption, "alg" MUST NOT be present and "enc" MUST be a content encryption algorithm from JSON Web Signature and Encryption Algorithms in {{JOSE-IANA}}.
```

Comment:
And this makes no sense as per my previous comment about making things required to be protected.

As such this section should be dispersed.

Reply:
Yes, removed section.